frontend/src/components/administration/administration.tsx

```diff
@@ -0,0 +1,26 @@
+import { User } from '@autoline/shared/common/types/types';
+import Box from '@mui/material/Box';
+import { useGetUsersQuery } from '@store/queries/users';
+
+import { UsersList } from './users/users-list';
+
+export const Administration = (): React.ReactElement => {
+  const { data, isLoading, isSuccess } = useGetUsersQuery();
+
+  return (
+    <Box sx={{ padding: '20px' }}>
+      <h1>Admin panel</h1>
+      {isLoading ? (
```

Comment:
Try to simplify this logic, very hard to understand what is going on here

Reply:
Thanks, fixed it.